TestObjectiveDropbox/Podfile.lock

```diff
@@ -9,7 +9,7 @@ EXTERNAL SOURCES:
     :path: ../
 
 SPEC CHECKSUMS:
-  ObjectiveDropboxOfficial: ad0d4cc85b64ebb3fce5139c04a54294a6b88fdb
+  ObjectiveDropboxOfficial: 37878160a4e4c7f3e3886d4d1e8ea90d12c05d67
```

Comment:
Is it intentional? Why we need to change podfile?

Reply:
Yeah this is just the test project, it's NBD.